packages/components/src/internal/components/domainproperties/TextFieldOptions.tsx

```diff
@@ -8,7 +8,7 @@ import { isFieldFullyLocked } from './propertiesUtil';
 import {
     DOMAIN_FIELD_CUSTOM_LENGTH,
     DOMAIN_FIELD_MAX_LENGTH,
-    DOMAIN_FIELD_SCALE,
+    DOMAIN_FIELD_SCALE, DOMAIN_FIELD_SCANABLE_OPTION,
```

Comment:
don't forget to run the linting on this PR: `yarn run lint-branch-fix`

Reply:
done